identity/app/controllers/RegistrationController.scala

```diff
@@ -46,8 +46,9 @@ class RegistrationController @Inject()( returnUrlVerifier : ReturnUrlVerifier,
 
     val idRequest = idRequestParser(request)
     val filledForm = registrationForm.bindFromFlash.getOrElse(registrationForm.fill("", "", "", "", "", true, false))
+    val socialEmailRemoved = request.getQueryString("fbEmailError").map(_ == "true")
```

Comment:
Can this be updated to support: 

https://profile.theguardian.com/signin?error=fbEmail

We will then have a generic way for trigger error messages on the page.